src/scripts/clipperUI/components/modeButton.tsx

```diff
@@ -23,18 +25,18 @@ class ModeButtonClass extends ComponentBase<{}, ModeButtonProps> {
 		if (this.props.selected) {
 			className += " selected";
 		}
-
 		let clipMode: string = ClipMode[this.props.myMode];
 		clipMode = clipMode[0].toLowerCase() + clipMode.slice(1);
 		let idName: string = clipMode + "Button";
 
 		return (
-			<a className={className} aria-selected={this.props.selected} role="button"
-				id={idName} title={this.props.tooltipText ? this.props.tooltipText : ""}
-				{...this.enableInvoke(this.buttonHandler, this.props.tabIndex) }>
-				<img className="icon" src={this.props.imgSrc} />
-				<span className="label buttonLabelFont"
-					style={Localization.getFontFamilyAsStyle(Localization.FontFamily.Regular)}>
+			<a
+				className={className} role="option" data-ariaModeButtonSet={Constants.AriaSet.modeButtonSet}
+				id={idName} title={this.props.tooltipText ? this.props.tooltipText : ""} aria-setsize={this.props["aria-setsize"]} aria-posinset={this.props["aria-posinset"]}
+				{...this.enableInvoke(this.buttonHandler, this.props.tabIndex)} tabIndex={this.props.tabIndex}
```

Comment:
tabIndex attribute is not needed. One job of `enableInvoke` is to set the tabIndex on the element.